app/controllers/attendees_controller.rb

```diff
@@ -0,0 +1,56 @@
+class AttendeesController < ApplicationController
+  before_action :set_attendee, only: [:show, :edit, :update, :destroy]
+  before_action :set_event
+
+  def index
+    @attendees = @event.attendees
+  end
+
+  def new
+    @attendee = @event.attendees.build
+  end
+
+  def create
+    @attendee = @event.attendees.build(attendee_params)
+
+    if @attendee.save
+      redirect_to Event.find(@attendee.event_id)
```

Comment:
It's better to use url_helper to create path and this way we can avoid event finder here.